src/base/projects/Browser.svelte

```diff
@@ -353,18 +392,18 @@
           {#await getAnchor}
             <Loading small margins />
           {:then anchor}
-            {#if anchor === commit}
+            {#if anchor?.includes(commit)}
               {#if commit === project.head}
                 <span class="anchor-widget anchor-latest">
                   <span class="anchor-label" title="{anchors}">anchored 🔒</span>
                 </span>
               {:else}
                 <span class="anchor-widget" on:click={() => navigateBrowser(project.head)}>
-                  <span class="anchor-label" title="{anchors}">anchored 🔒</span>
+                  <span class="anchor-label" title="{anchors}">{getAnchorIndexByCommit(anchor, commit)} anchor behind latest 🔒</span>
```

Comment:
Let's leave the copy here as "anchored" for now. I need to think a little bit more about how we should differentiate this case.

Reply:
To avoid misunderstandings, you mean that all anchors should simply show up as `anchored 🔒` ?

Reply:
Oh, no I mean that if a commit was anchored in the past, ie. `anchors.include(commit)`, it should be marked as `anchored`. Before, this would be `not anchored`.

Reply:
Hmm ok, maybe we can name it `anchored (stale) 🔒` and eventually give it a different background color, to differentiate a past anchor and the current one?

Reply:
Ok how about `stale 🔒` and `latest 🔒`? The lock shows that it's anchored, while the text tells you whether it's an old anchor or the latest?

Reply:
maybe latest has the key as well:  `stale 🔒` and `latest 🔐`

Reply:
Okay yeah I'm fine with that :+1:

Reply:
Okay I'm having another issue now with this PR, how do we want to handle the interaction with this button..
- commit == head && latest anchor, it should show anchor(latest) and not lead anywhere
- commit != head && anchor == latest, it should lead to the latest commit (head), current behaviour.
- commit != head && anchor != latest, should it lead to latest anchor or latest head?
- commit != head && !anchor, where should it lead?